pkg/storegateway/series_refs.go

```diff
@@ -562,14 +563,9 @@ func openBlockSeriesChunkRefsSetsIterator(
 		minTime,
 		maxTime,
 	)
-
-	limitingIterator := newLimitingSeriesChunkRefsSetIterator(
-		loadingIterator,
-		chunksLimiter,
-		seriesLimiter,
-	)
-
-	return limitingIterator, nil
+	iterator = newDurationMeasuringIterator[seriesChunkRefsSet](iterator, metrics.iteratorLoadDurations.WithLabelValues("index_load"))
```

Comment:
[nit] Given it's called "chunks_load" for chunks, I would just call it "series_load" here. Regardless from where data is loaded, all in all we have to load series and chunks.

```suggestion
	iterator = newDurationMeasuringIterator[seriesChunkRefsSet](iterator, metrics.iteratorLoadDurations.WithLabelValues("series_load"))
```

Reply:
i missed this. thanks for catching it